include/SDL_gpu_shadercross.h

```diff
@@ -0,0 +1,254 @@
+/*
+  Simple DirectMedia Layer Shader Cross Compiler
+  Copyright (C) 2024 Sam Lantinga <slouken@libsdl.org>
+
+  This software is provided 'as-is', without any express or implied
+  warranty.  In no event will the authors be held liable for any damages
+  arising from the use of this software.
+
+  Permission is granted to anyone to use this software for any purpose,
+  including commercial applications, and to alter it and redistribute it
+  freely, subject to the following restrictions:
+
+  1. The origin of this software must not be misrepresented; you must not
+     claim that you wrote the original software. If you use this software
+     in a product, an acknowledgment in the product documentation would be
+     appreciated but is not required.
+  2. Altered source versions must be plainly marked as such, and must not be
+     misrepresented as being the original software.
+  3. This notice may not be removed or altered from any source distribution.
+*/
+
+#ifndef SDL_GPU_SHADERCROSS_H
+#define SDL_GPU_SHADERCROSS_H
+
+#ifdef _WIN32
+#define SHADERCROSSAPI __declspec(dllexport)
+#define SHADERCROSSCALL __cdecl
+#else
+#define CRAMAPI
+#define CRAMCALL
+#endif
+
+#include <SDL3/SDL_gpu.h>
+
+#ifndef SDL_GPU_SHADERCROSS_SPIRVCROSS
+#define SDL_GPU_SHADERCROSS_SPIRVCROSS 1
+#endif /* SDL_GPU_SHADERCROSS_SPIRVCROSS */
+
+#ifndef SDL_GPU_SHADERCROSS_HLSL
+#define SDL_GPU_SHADERCROSS_HLSL 1
+#endif /* SDL_GPU_SHADERCROSS_HLSL */
+
+#ifndef SDL_GPU_SHADERCROSS_EXPORT
+#define SDL_GPU_SHADERCROSS_EXPORT
+#endif
```

Comment:
This is superseded by `SHADERCROSSAPI`. (the original intent was for you to define `SDL_GPU_SHADERCROSS_EXPORT` as `__declspec(dllexport)` if needed)